flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/ZooKeeperLeaderElectionTest.java

```diff
@@ -438,13 +441,20 @@ void testLeaderShouldBeCorrectedWhenOverwritten() throws Exception {
             boolean dataWritten = false;
 
             final String connectionInformationPath =
-                    leaderElectionDriver.getConnectionInformationPath();
+                    ZooKeeperUtils.generateConnectionInformationPath(CONTENDER_ID);
 
             while (!dataWritten) {
                 anotherCuratorFrameworkWrapper
                         .asCuratorFramework()
                         .delete()
                         .forPath(connectionInformationPath);
+                System.out.println(
```

Comment:
?

Reply:
That was accidentally left. I removed it as part of the test.